src/commands/yarn_install.yml

```diff
@@ -9,6 +9,10 @@ parameters:
     description: The path to the yarn cache folder.  Defaults to /tmp/yarn
     type: string
     default: "/tmp/yarn"
+  yarn_install_directory:
+    description: The working directory to run install at. Defaults to current directory
+    type: string
+    default: ""
```

Comment:
Shouldn't the default be `"."` or an absolute path to the current directory?

Reply:
It _could_ `"."` or `"./"` but by not passing anything it allows `yarn` to do its default logic. I'm not sure which is preferred here 🤔 

I think I could also do a conditional on `yarn_install_directory` to omit `--cwd` if its not an empty string?

Reply:
I have little insights behind this orb, but I would refrain from using the yarn's default logic for empty strings. Should that change, this orb will break for most of the users (as it's the default). Both solutions you mentioned sounds good to me.

Reply:
I'll add the conditional check! That seems safest and most backwards compatible to me 😊 Thanks you for the reviews! I'll get that commit added once my kid goes down for his next nap 😛